File: internal/model/burnpair/uniswap_pair.go
package burnpair

// UniswapPair Uniswap交易对实体
type UniswapPair struct {
	PairAddress     string `gorm:"primary_key;column:pair_address;type:varchar(66)" json:"pairAddress"`
	TokenAddress    string `gorm:"column:token_address;type:varchar(66)" json:"tokenAddress"`
	CurrencyAddress string `gorm:"column:currency_address;type:varchar(66)" json:"currencyAddress"`
	TokenFirst      int16  `gorm:"column:token_first;default:0" json:"tokenFirst"`
	FactoryAddress  string `gorm:"column:factory_address;type:varchar(66)" json:"factoryAddress"`
	PairCreateBlock uint64 `gorm:"column:pair_create_block;default:0" json:"pairCreateBlock"`
	UpdatedBlock    uint64 `gorm:"column:updated_block" json:"updatedBlock"`
	ValueState      int    `gorm:"column:value_state" json:"valueState"`
	ValueInUsd      int    `gorm:"column:value_in_usd" json:"valueInUsd"`
	CheckState      *int   `gorm:"column:check_state" json:"checkState"`
	CodeGot         int16  `gorm:"column:code_got;default:0" json:"codeGot"`
	Deleted         int    `gorm:"column:deleted;default:0" json:"deleted"`
}

// TableName 设置表名
func (UniswapPair) TableName() string {
	return "uniswap_pair"
}

File: src/main/java/beholder_daemon/entity/burnPair/UniswapPair.java
package beholder_daemon.entity.burnPair;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name = "uniswap_pair")
/*@SQLInsert(sql = "INSERT IGNORE INTO uniswap_pair (" +
    "check_state, currency_address, deleted, factory_address, " + 
    "pair_create_block, token_address, token_first, updated_block, " +
    "value_in_usd, value_state, pair_address" +
    ") VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)")*/
public class UniswapPair {

    @Id
    @Column(name = "pair_address", nullable = false, length = 66)
    private String pairAddress;

    @Column(name = "token_address", nullable = false, length = 66)
    private String tokenAddress;

    @Column(name = "currency_address", nullable = false, length = 66)
    private String currencyAddress;

    @Column(name = "token_first", nullable = false)
    private Short tokenFirst = 0;

    @Column(name = "factory_address", nullable = false, length = 66)
    private String factoryAddress;

    @Column(name = "pair_create_block", nullable = false)
    private Long pairCreateBlock = 0L;

    @Column(name = "updated_block", nullable = false)
    private Long updatedBlock;

    @Column(name = "value_state")
    private Integer valueState;

    @Column(name = "value_in_usd")
    private Integer valueInUsd;

    @Column(name = "check_state")
    private Integer checkState;

    @Column(nullable = false)
    private Short codeGot = 0;

    @Column(name = "deleted", nullable = false)
    private Integer deleted = 0;
}